src/main/java/com/microsoft/azure/servicebus/jms/ServiceBusJmsConnectionFactorySettings.java

```diff
@@ -12,11 +12,18 @@
 public class ServiceBusJmsConnectionFactorySettings {
     private static Map<String, String> DefaultConfigurationOptions = getDefaultConfigurationOptions();
     
+    static final String Vendor = "com.microsoft";
```

Comment:
Vendor will not change why separate?

Reply:
Just so we can avoid repeating when we add new constants and reuse the Vendor part of the field.